tests/test_add_mds_to_cluster.py

```diff
@@ -0,0 +1,144 @@
+                size: 3
+          # The metadata service (mds) configuration
+          metadataServer:
+            # The number of active MDS instances
+            activeCount: {active_count}
+            # Whether each active MDS instance will have an active standby
+            # with a warm metadata cache for faster failover.
+            # If false, standbys will be available, but will not have
+            # a warm cache.
+            activeStandby: true
+            # The affinity rules to apply to the mds deployment
+            placement:
+            #  nodeAffinity:
+            #    requiredDuringSchedulingIgnoredDuringExecution:
+            #      nodeSelectorTerms:
+            #      - matchExpressions:
+            #        - key: role
+            #          operator: In
+            #          values:
+            #          - mds-node
+            #  tolerations:
+            #  - key: mds-node
+            #    operator: Exists
+            #  podAffinity:
+            #  podAntiAffinity:
+            # A key/value list of annotations
+            annotations:
+            #  key: value
+            resources:
+            # The requests and limits set here, allow the filesystem MDS
+            # Pod(s) to use half of one CPU core and 1 gigabyte of memory
+            #  limits:
+            #    cpu: "500m"
+            #    memory: "1024Mi"
+            #  requests:
+            #    cpu: "500m"
+            #    memory: "1024Mi"
+        """
+    )
+
+    fs_dict = yaml.safe_load(fs_body)
+    log.info(f"Creating a new Ceph FileSystem")
+    ret = client.create(body=fs_dict)
+    set_trace()
+    log.info(f"Return value is:\n{ret}")
+    return ret
+
+
+def modify_fs(client, dict_to_modify, new_active_count):
+    """
+    """
+    dict_to_modify['spec']['metadataServer']['activeCount'] = new_active_count
+    log.info(f"Change the active_count to {new_active_count}")
+    ret = client.replace(body=dict_to_modify)
+    log.info(f"The updated return value is:\n{ret}")
+    return ret
+
+
+def delete_fs(client, fs_name):
+    """
+    """
+    log.info(f"Deleting the file system")
+    ret = client.delete(
+        name=fs_name, namespace=defaults.ROOK_CLUSTER_NAMESPACE
+    )
+    if ret.status == "Success":
+        return True
+    return False
+
+
+def verify_fs_exist(client, fs_name):
+    """
+    """
+    try:
+        log.info(f"Verifying if the filesystem exist or not")
+        client.get(name=fs_name, namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+    except exceptions.ApiException:
+        log.info(f"Ceph FS {fs_name} doesn't exist")
+        return False
+    log.info(f"Ceph FS {fs_name} exist")
+    return True
+
+
+def run(**kwargs):
+    """
+    A simple function to exercise a resource creation through api-client
+    """
+    fs_name = 'my-cephfs'
+    active_count = '1'
+    new_active_count = '2'
+    ceph_fs_resource = dyn_client.resources.get(
+        api_version='v1', kind='CephFilesystem'
```

Comment:
@RazTamir 
You are using api_version='v1' , Don't we have to use ceph.rook.io/v1 ? OR it doesn't matter at all ?

I am still not familiar with api namespaces and rook's interaction in kubernets, just a hunch?Can you confirm it once.

Reply:
It doesn't seem like a problem to use only 'v1'.
I don't know what is the difference